debian/control

```diff
@@ -4,15 +4,13 @@ Priority: optional
 Maintainer: Derp Ston <derpston+packaging@sleepygeek.org>
 Build-Depends:
     debhelper (>= 6.0.0),
-    python-support (>= 0.90)
 Standards-Version: 3.9.1
 
 Package: python-graphiteudp
 Architecture: all
 Depends:
     ${python:Depends},
     ${misc:Depends},
-    python-socketcache
```

Comment:
Not sure about removing this, pretty sure it's still a dep!

Reply:
ah yes, that also depends on `python-support` so is broken on 16.04, I'm going to need to also perform this change on `python-socketcache` so it doesn't use that package before merging this